.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+build
+local_test
```

Comment:
This line should have a newline.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
@ValeraS If you can just fix this one line I am ready to merge this. Thanks.

Reply:
@tchaffee it's done.